src/main/java/TaskList.java

```diff
@@ -0,0 +1,106 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> todoList = new ArrayList<>();
+
+    public TaskList (ArrayList<Task> taskList) {
+        todoList = taskList;
+    }
+
+    public int getSize(){
+        return todoList.size();
+    }
+
+    /**
+     * Returns the number of unfinished tasks in the todoList
+     * @return number of unfinished tasks
+     */
+    public int getUndoneCount() {
+        int undoneCount = 0;
+        for (Task t:todoList){
+            if (!t.checkDone())
+                undoneCount++;
+        }
+
+        return undoneCount;
+    }
+
+    /**
+     * Create a new TodoTask, add it to the totoList and returns it.

```

Comment:
Should this comment start with "Creates" or "Create"? I noticed this issue in several other places too.